Awesome/index2.html

```diff
@@ -0,0 +1,317 @@
+              </div>
+            </div>
+          </div>
+          <div class="pricing__card_hover">
+            <div class="card__shadow">
+            <div class="pricing__card-top">
+              <div class="pricing__card-title">
+                <p>Optima</p>
+              </div>
+              <div class="pricing__card-price">
+                <p>$20</p>
+              </div>
+              <div class="pricing__card-subtitle">
+                <p>Free for life</p>
+              </div>
+            </div>
+            <div class="pricing__card-list">
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+            </div>
+          </div>
+          </div>
+          <div class="pricing__card">
+            <div class="pricing__card-top">
+              <div class="pricing__card-title">
+                <p>Pro</p>
+              </div>
+              <div class="pricing__card-price">
+                <p>$30</p>
+              </div>
+              <div class="pricing__card-subtitle">
+                <p>Free for life</p>
+              </div>
+            </div>
+            <div class="pricing__card-list">
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+              <div class="pricing__card-list-item">
+                <p>1 gb of space</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="customers">
+        <div class="pricing__tittle-wrapper">
+          <h2 class="awesome__title">what our customers are saying</h2>
+        </div>
+        <div class="awesome__line"></div>
+        <h5 class="customers__subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h5>
+        <div class="comments">
+          <div class="comment__block1">
+              <div class="quotation">
+                <div class="quotation__text">
```

Comment:
Why do you need div around your p element?

Reply:
I thought, that style won't be applied just to p element. Fixed, the div was deleted.